docs/guides/external-node/10_attestator_committee.md

```diff
@@ -0,0 +1,41 @@
+# Attester Committee
+
+## Overview
+
+Attester committee is a subset of ZKSync nodes. 
+After each l1 batch execution, participating nodes sign the execution result and send it back to the main node.
+
+The ultimate goal is to make L1 commit operation contingent on such signatures.
+This will improve the security and finality guarantees: 
+having these signatures on L1 shows that additional actors executed the corresponding blocks - 
+and ended up with the same state root hash.
+
+## Current State
+
+Before we start the L1 integration,
+we want to make sure we are able to consistently reach the quorum and collect the signatures in a timely manner. 
+Currently the main node just stores the signatures in the local DB (`l1_batches_consensus` table).
+
+We run a (temporary) PoA network of attesters where the Main Node administrator defines the committee for every L1 batch.
+We'll be developing incentives and potential tokenomics later on - please stay tuned.
+
+We are looking for participants to this network.
```

Comment:
We should make it clearer that attesters are unpaid. Otherwise people might expect some later airdrop for participating.